tests/gis_tests/test_geoip2.py

```diff
@@ -230,6 +219,30 @@ class GeoIP2Test(GeoLite2Test):
     """Non-free GeoIP2 databases are supported."""
 
 
+@skipUnless(HAS_GEOIP2, "GeoIP2 is required.")
+@override_settings(
+    GEOIP_CITY="dbip-city-lite-test.mmdb",
+    GEOIP_COUNTRY="dbip-country-lite-test.mmdb",
+)
+class DBIPLiteTest(GeoLite2Test):
+    """DB-IP Lite databases are supported."""
+
+    @classproperty
+    def expected_city(cls):
+        # Override result where values differ to other databases.
+        return super().expected_city | {
```

Comment:
Could this just be...?
```suggestion
    expected_city = GeoLite2Test.expected_city | {
```

Reply:
Yeah, it could.